aspnetcore/host-and-deploy/iis/troubleshoot.md

```diff
@@ -45,7 +57,8 @@ Access the Application Event Log:
 
 1. Open the Start menu, search for **Event Viewer**, and then select the **Event Viewer** app.
 1. In **Event Viewer**, open the **Windows Logs** node.
-1. Select **Application** to open the Application Event Log. Search for errors associated with the failing app.
+1. Select **Application** to open the Application Event Log.
+1. Search for errors associated with the failing app. Errors have a value of *IIS Express AspNetCore Module* in the *Source* column.
```

Comment:
IIS or IIS Express

Reply:
So that source column value can be either "IIS AspNetCore Modue" or "IIS Express AspNetCore Module" ... I see.